.gitignore

```diff
@@ -7,3 +7,5 @@ __pycache__
 /node_modules
 /project/settings/local.py
 /static
+/venv
+/.idea
```

Comment:
Found the JetBrains user. 🙂

Please don't add stuff to `.gitignore` unless it's artifacts created by the project itself. IDE files should be exlcuded by your `.gitignore_global` file: https://docs.github.com/en/github/getting-started-with-github/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Doh, thanks for the info.